### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by megan07**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
 ## 4.44.0 (Unreleased)
 
+FEATURES:
+* **New Resource:** `google_alloydb_instance` ([#12981](https://github.com/hashicorp/terraform-provider-google/pull/12981))
+* **New Resource:** `google_compute_network_firewall_policy` ([#12969](https://github.com/hashicorp/terraform-provider-google/pull/12969))
+* **New Resource:** `google_compute_region_network_firewall_policy` ([#12969](https://github.com/hashicorp/terraform-provider-google/pull/12969))
+* **New Resource:** `google_firebase_hosting_site` ([#12960](https://github.com/hashicorp/terraform-provider-google/pull/12960))
+* **New Resource:** `google_kms_crypto_key_version` ([#12926](https://github.com/hashicorp/terraform-provider-google/pull/12926))
+* **New Resource:** `google_storage_transfer_agent_pool` ([#12945](https://github.com/hashicorp/terraform-provider-google/pull/12945))
+* **New Resource:** `identity_platform_project_default_config` ([#12977](https://github.com/hashicorp/terraform-provider-google/pull/12977))
```
Comment (slevenick) [inline CHANGELOG.md:10]: google_?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+
+IMPROVEMENTS:
+* cloudidentity: made security label settable by making labels updatable on `google_cloud_identity_group` ([#12943](https://github.com/hashicorp/terraform-provider-google/pull/12943))
```
Comment (slevenick) [inline CHANGELOG.md:13]: added update support for security label?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+* compute: updated machine_type field in compute instance resource to accept short name. ([#12965](https://github.com/hashicorp/terraform-provider-google/pull/12965))
+* container: Added support for Shielded Instance configuration for node auto-provisioning to `google_container_cluster` ([#12930](https://github.com/hashicorp/terraform-provider-google/pull/12930))
```
Comment (slevenick) [inline CHANGELOG.md:15]:
```suggestion
* container: added support for Shielded Instance configuration for node auto-provisioning to `google_container_cluster` ([#12930](https://github.com/hashicorp/terraform-provider-google/pull/12930))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,36 @@
...
+* container: added support for additional values `APISERVER`, `CONTROLLER_MANAGER`, and `SCHEDULER` in `google_container_cluster.monitoring_config` ([#12978](https://github.com/hashicorp/terraform-provider-google/pull/12978))
+* dlp: added fields `rows_limit`, `rows_limit_percent`, and `sample_method` to `big_query_options` in `google_data_loss_prevention_job_trigger` ([#12980](https://github.com/hashicorp/terraform-provider-google/pull/12980))
+* dlp: added pubsub action to `google_data_loss_prevention_job_trigger` ([#12929](https://github.com/hashicorp/terraform-provider-google/pull/12929))
+* sql: added `query_plan_per_minute` field to `insights_config` in `google_sql_database_instance` resource ([#12951](https://github.com/hashicorp/terraform-provider-google/pull/12951))
+* vertexai: added fields to `vertex_ai_featurestore_entitytype` to support feature value monitoring ([#12983](https://github.com/hashicorp/terraform-provider-google/pull/12983))
+
+BUG FIXES:
+* appengine: Fixed permadiff on serviceaccount for 'google_app_engine_flexible_app_version' ([#12982](https://github.com/hashicorp/terraform-provider-google/pull/12982))
```
Comment (slevenick) [inline CHANGELOG.md:25]:
```suggestion
* appengine: fixed permadiff on serviceaccount for 'google_app_engine_flexible_app_version' ([#12982](https://github.com/hashicorp/terraform-provider-google/pull/12982))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,36 @@
...
+* bigtable: updated the error handling logic to remove the resource on resource not found error only ([#12953](https://github.com/hashicorp/terraform-provider-google/pull/12953))
+* billingbudget: fixed a bug where `budget_filter.credit_types_treatment` in `google_billing_budget` resource was not updating. ([#12947](https://github.com/hashicorp/terraform-provider-google/pull/12947))
+* cloudids: fixed `endpoint_forwarding_rule` and `endpoint_ip` attributes for `google_cloud_ids_endpoint` ([#12957](https://github.com/hashicorp/terraform-provider-google/pull/12957))
+* compute: fixed perma-diff on `google_compute_disk` for new amd64 images ([#12961](https://github.com/hashicorp/terraform-provider-google/pull/12961))
+* compute: made `target_https_proxy` possible to set `ssl_certificates` and `certificate_map` at the same time ([#12950](https://github.com/hashicorp/terraform-provider-google/pull/12950))
```
Comment (slevenick) [inline CHANGELOG.md:30]: made google_compute_...?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,36 @@
...
+* dns: fixed apply failure when `description` is set to empty string on `google_dns_managed_zone` ([#12948](https://github.com/hashicorp/terraform-provider-google/pull/12948))
+* vertex_ai:  made google_vertex_ai_featurestore_entitytype always use regional endpoint corresponding to parent's region ([#12959](https://github.com/hashicorp/terraform-provider-google/pull/12959))
```
Comment (slevenick) [inline CHANGELOG.md:32]:
```suggestion
* vertex_ai:  made `google_vertex_ai_featurestore_entitytype` always use regional endpoint corresponding to parent's region ([#12959](https://github.com/hashicorp/terraform-provider-google/pull/12959))
```

**Comment by c2thorn** [comment]
superseded by https://github.com/hashicorp/terraform-provider-google/pull/13081

**Pull request closed without merging**